tmk_core/common/chibios/wait.c

```diff
@@ -14,76 +14,33 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-#ifndef __OPTIMIZE__
-#    pragma message "Compiler optimizations disabled; wait_cpuclock() won't work as designed"
+#include <ch.h>
+#include <hal.h>
+
+#include "_wait.h"
+
+#ifdef WAIT_US_TIMER
+static thread_t *main_thread = NULL;
+static const GPTConfig gpt_cfg = { 100000, NULL, 0, 0 }; /* 1MHz timer, no callback */
```

Comment:
Should be either one of:
```suggestion
static const GPTConfig gpt_cfg = { 100000, NULL, 0, 0 }; /* 100kHz timer, no callback */
```
**OR**
```suggestion
static const GPTConfig gpt_cfg = { 1000000, NULL, 0, 0 }; /* 1MHz timer, no callback */
```

Reply:
I've changed this to 1MHz as intended